vscode/package.json

```diff
@@ -123,10 +125,11 @@
             "plsql",
             "postgresql",
             "redshift",
+            "singlestoredb",
             "spark",
             "sqlite",
             "trino",
-            "tsql"
+            "transactsql"
```

Comment:
I'd suggest not removing tsql, but keeping both.

When you remove it, it's likely the config will break for anyone who uses this value. Though I don't really know how VSCode will behave in this case. Will switch to the default value? Will it keep the old value regardless it being invalid according to this config? Something else?